react/wbooks/src/app/components/NavBar/styles.module.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../../scss/colors';
+
+$login-main-component-width: 835px;
+$navbar-height: 70px;
+
+.logout {
+  font-size: 16px;
+  font-weight: 700;
+  margin-left: auto;
```

Comment:
is this necessary?

Reply:
Yes! Is to put the logout label at the rigth side of the navbar. I tried many things and this result the prettiest

Reply:
@raparicio6 You could flex, with `align-self: end`